webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -172,7 +154,6 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
             isCloudTrial,
             hadPrevCloudTrial,
             isPaidSubscription,
-            minimumSKURequiredForFeature,
         } = this.props;
 
         const canRequestCloudFreeTrial = isCloud && !isCloudTrial && !hadPrevCloudTrial && !isPaidSubscription;
```

Comment:
Should we remove all code paths where this is true? It seems we can no longer request a cloud trial in product